guava-recent-unit-tests/build.gradle

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2022. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+plugins {
+    id 'java-library'
+    id 'nullaway.jacoco-conventions'
```

Comment:
If we want coverage from tests in this module to be included as part of the overall code coverage report (sent to Coveralls), we should add an `implementation` dependence on this module here:

https://github.com/uber/NullAway/blob/4cbd78106d087ee0c9c9dad73729a6589a222db8/code-coverage-report/build.gradle#L90-L95

Reply:
Done. Also making sure this builds on all JDKs we support.